index.js

```diff
@@ -2,16 +2,7 @@
 const _ = require('lodash');
 
 exports.register = (server, config, next) => {
-  // set up logging:
-  const log = (tags, content) => {
-    if (!config.verbose) {
-      return;
-    }
-    server.log(tags, content);
-  };
-  log(['hapi-strategy-loader', { msg: 'Loading strategy plugins' }]);
   _.forIn(config.strategies, (value, name) => {
-    log(['hapi-strategy-loader'], { message: 'strategy loaded', strategy: name, options: value });
```

Comment:
this log should stay.  it was just loading and loaded (didn't really provide any value)